frontend/src/app/offers/create-offer/create-offer.component.ts

```diff
@@ -137,4 +137,25 @@ export class CreateOfferComponent implements OnInit, OnDestroy {
     const input = this.form.get(inputStringId);
     return !input.valid && input.touched || (this.error && !input.valid);
   }
+
+  areDatesValid(form: FormGroup):  {[key: string]: boolean}  {
+    const startedAt = form.get('startedAt').value;
+    const actionOngoing = form.get('actionOngoing').value;
+    const finishedAt = form.get('finishedAt').value;
+    const constantCoop = form.get('constantCoop').value;
+    let hasAnyError = false;
+    const validationError = {};
+
+    if (startedAt && actionOngoing) {
+      hasAnyError = true;
+      validationError['startedAtError'] = true;
+    }
+
+    if (finishedAt && constantCoop) {
+      hasAnyError = true;
+      validationError['finishedAtError'] = true;
+    }
+
```

Comment:
Actually, you could just do here `const hasAnyError = Object.keys(validationErrors).length > 0` instead of repeating it above.